### ISSUE TITLE
Time-dependent goal-oriented mesh adaptation demo

### ISSUE THREAD

**Issue by jwallwork23** [mesh-adaptation/goalie#28]
Parent: pyroteus/pyroteus#94.

**Comment by ddundo** [comment]
@jwallwork23 could this demo be bubble shear? I pretty much have it ready to go. I'd be keen to get this up since I guess it would also be a good test case for running in parallel.

**Comment by jwallwork23** [comment]
> @jwallwork23 could this demo be bubble shear? I pretty much have it ready to go. I'd be keen to get this up since I guess it would also be a good test case for running in parallel.

That'd be great, thanks @ddundo. Will assign this to you.

**Comment by ddundo** [comment]
I see now that there is an open issue about what we were discussing after the meeting: https://github.com/pyroteus/goalie/issues/73. I am putting this slightly on hold for now.
